composer.json

```diff
@@ -27,12 +27,20 @@
         "mksanitizedparameters": "self.version",
         "typo3-ter/mksanitizedparameters": "self.version"
     },
+    "repositories": [{
```

Comment:
This should be resolved before we merge that PR

Reply:
Of course. Incidentally Sascha Egerer commented your PR https://github.com/sascha-egerer/phpstan-typo3/pull/10.

Reply:
Yeah, but this PR should be merged first. Using a fork stinks :P 
I will try push my PR forwards.